tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+from os.path import abspath, dirname, join, realpath
+import sys
+
+sys.path.append(abspath(join(dirname(realpath(__file__)), '..', 'securedrop')))
```

Comment:
Please add a comment to the code explaining why this is necessary - note I understand this is here because you moved the tests directory and you don't want to modify the imports in the tests, but it might not be straightforward to someone looking at the code for the first time